Source/Particles/MultiParticleContainer.cpp

```diff
@@ -743,11 +751,209 @@ void MultiParticleContainer::InitQED ()
         if(pc->has_quantum_sync()){
             pc->set_quantum_sync_engine_ptr
                 (shr_p_qs_engine);
+            someone_has_quantum_sync = true;
         }
         if(pc->has_breit_wheeler()){
             pc->set_breit_wheeler_engine_ptr
                 (shr_p_bw_engine);
+            someone_has_breit_wheeler = true;
         }
     }
+
+    if(someone_has_quantum_sync)
+        InitQuantumSync();
+
+    if(someone_has_breit_wheeler)
+        InitBreitWheeler();
+
+}
+
+void MultiParticleContainer::InitQuantumSync ()
+{
+    bool generate_table;
+    PicsarQuantumSynchrotronCtrl ctrl;
+    std::string filename;
+    std::tie(generate_table, filename, ctrl) = ParseQuantumSyncParams();
+
+    //Only temporary for test purposes, will be removed
```

Comment:
In general, if QED effects are enabled lookup tables should be either generated (this requires Boost) or loaded from disk. The initialization of the optical depth is the only step which does not require QED tables, so I've introduced this setting to be able to add a test without having the lookup tables. 
We could add some default tables (100s kB binary files) directly in the git repository, but since they are likely to change several times in this phase, I would like to delay that step. I would like to discuss the option of using `git-annex` (this would allow me to change the lookup tables as many times as I want without polluting the git repository). 

So, basically, the answer to your question is: if we want to keep a test of the optical depth initialization, this flag should probably stay there until we find a definitive solution for the lookup tables.

Reply:
Ok agreed, let's keep this.